open_api/amoc_rest/rebar.config

```diff
@@ -0,0 +1,6 @@
+{deps, [
+    {cowboy, {git, "https://github.com/ninenines/cowboy.git", {tag, "2.5.0"}}},
```

Comment:
That's quite an old version of cowboy, cat it work with `2.7.0`?

Reply:
i think we can safely upgrade it, will check that.